modules/ducktests/tests/checks/utils/check_ignite_spec.py

```diff
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Checks Spec class that describes config and command line to start Ignite-aware service.
+"""
+from unittest.mock import Mock
+
+from ignitetest.services.utils.ignite_spec import IgniteApplicationSpec
+from ignitetest.utils.ignite_test import JFR_ENABLED
+
+
+def mock_service():
+    """
+    Create mock of service.
+    """
+    service = Mock()
+    service.log_dir = ''
+    service.persistent_root = ''
+    service.context.globals = {"cluster_size": 1}
+
+    return service
+
+
+class CheckIgniteSpec:
```

Comment:
Actually, class is redundant. You can just use test functions in package, it is ok for pytest

Reply:
```
@pytest.fixture
def mock_service():
    """
    Create mock of service.
    """
    service = Mock()
    service.log_dir = ''
    service.persistent_root = ''
    service.context.globals = {"cluster_size": 1}

    return service

def check_default_options__are_used__if_jvm_opts_is_not_passed(mock_service):
   ....
```

Reply:
done